tests/test_api.py

```diff
@@ -6,11 +6,21 @@
 import sys
 import io
 from rsconnect.exception import RSConnectException
+from rsconnect.metadata import CURRENT_APP_STORE_VERSION
```

Comment:
The testing strategy might be better if we have a scaffold of tests for each version. That way we can just add new tests when version advances instead of needing to modify or delete existing test. We also want to have tests for older versions.